docs/provisioning/silo_vnet_existingstorage.md

```diff
@@ -0,0 +1,93 @@
+# Create a silo with compute behind a vnet accessing your existing storage account
+
+This tutorial applies in the case you want to create a compute to process data from an existing storage account located in the same tenant (different subscription or resource group).
+
+## Table of contents
+
+- [Prerequisites](#prerequisites)
+- [Important: understand the design](#important-understand-the-design)
+- [Create a compute and storage pair for the silo](#need-support)
+  - [Option 1: one click deployment](#option-1-one-click-deployment)
+  - [Option 2: using az cli](#option-2-using-az-cli)
+- [Set required permissions](#set-required-permissions)
+
+## Prerequisites
+
+To run these deployment options, you first need:
+- an existing Azure ML workspace (see [cookbook](README.md))
+- an existing private DNS zone for storage, named `privatelink.blob.core.windows.net` (see below)
+- have permissions to create resources, set permissions, and create identities in this subscription (or at least in one resource group),
+  - Note that to set permissions, you typically need _Owner_ role in the subscription or resource group - _Contributor_ role is not enough. This is key for being able to _secure_ the setup.
+- Optional: [install the Azure CLI](https://learn.microsoft.com/en-us/cli/azure/install-azure-cli).
+
+
+> **To create a private DNS zone**  
+> If you don't already have one, you will need to manually create a private DNS zone for the storage account and compute of this pair.  
+> To do that, go to the Azure portal, and in the resource group of your AzureML workspace, create a new private DNS zone named `privatelink.blob.core.windows.net`.  
+> You only need one unique zone for all the pairs you create (both orchestrator and silos). All private DNS entries will be written in that single zone.
+
+## Important: understand the design
+
+The design we used for this tutorial is identical to [a silo provisioned with a new storage](./silo_vnet_newstorage.md#important-understand-the-design). What is different in this case is that we do not provision the storage, so we rely on the previously configured storage account.
+
+It is important in this case that you set the following on your existing storage account:
+- make sure the storage account is in the same tenant as the AzureML workspace
```

Comment:
nit: maybe put **in the same tenant** in bold?

Reply:
done, thx